File: Examples/CSharp/Worksheets/PageSetupFeatures/ManagePaperSize.cs
using System.IO;
using Aspose.Cells;
using System;

namespace Aspose.Cells.Examples.CSharp.Worksheets.PageSetupFeatures
{
    public class ManagePaperSize
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            // Instantiating a Workbook object
            Workbook workbook = new Workbook();

            // Accessing the first worksheet in the Excel file
            Worksheet worksheet = workbook.Worksheets[0];

            // Setting the paper size to A4
            worksheet.PageSetup.PaperSize = PaperSizeType.PaperA4;

            // Save the Workbook.
            workbook.Save(dataDir + "ManagePaperSize_out.xls");
            // ExEnd:1
        }
    }
}

File: Examples/VisualBasic/Worksheets/PageSetupFeatures/ManagePaperSize.vb
Imports System.IO
Imports Aspose.Cells

Namespace Worksheets.PageSetupFeatures
    Public Class ManagePaperSize
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Instantiating a Workbook object
            Dim workbook As New Workbook()

            ' Accessing the first worksheet in the Excel file
            Dim worksheet As Worksheet = workbook.Worksheets(0)

            ' Setting the paper size to A4
            worksheet.PageSetup.PaperSize = PaperSizeType.PaperA4

            ' Save the Workbook.
            workbook.Save(dataDir & Convert.ToString("ManagePaperSize_out.xls"))
            ' ExEnd:1
        End Sub
    End Class
End Namespace